apps/web/modules/bookings/views/bookings-single-view.test.tsx

```diff
@@ -11,6 +11,13 @@ import { HeadSeo } from "@calcom/ui";
 
 import Success from "./bookings-single-view";
 
+vi.mock("@calcom/lib/constants", async () => {
+  return {
+    WEBAPP_URL: "http://localhost:3000",
+    APP_NAME: "Cal.com",
+  };
+});
+
```

Comment:
WEBAPP_URL should be http://app.cal.local:3000 here

Reply:
Made the required changes.